css/src/components/form/checkbox.scss

```diff
@@ -88,9 +99,24 @@ $checkbox-spacing: $spacer-3 !default;
 		}
 
 		&[disabled] + .checkbox-check {
```

Comment:
I can't really see why, but we've restarted that iOS checkbox bug, where on iOS if we hold our finger over a disabled checkbox is looks checked permanently because fo the hover style ...

Reply:
Oh, I think it's a separate bug. I just checked on my iphone and looks like if you're clicking on checkbox it turns into "checked" state which is expected. And if you click again, checkbox is not blue but still shows the check icon. And it stays until you click on any other checkbox. It might be connected, so I'll create a bug and link it here shortly.

Reply:
Actually I found what caused the issue so finally pushed the update! Please take a look again